Forms/GridField/GridField.php

```diff
@@ -307,12 +307,12 @@ public function getState($getData = true) {
 	 */
 	public function FieldHolder($properties = array()) {
 		Requirements::css(THIRDPARTY_DIR . '/jquery-ui-themes/smoothness/jquery-ui.css');
-		Requirements::css(FRAMEWORK_DIR . '/client/dist/styles/GridField.css');
+		Requirements::css(trim(FRAMEWORK_DIR . '/client/dist/styles/GridField.css', '/'));
```

Comment:
That'll lead to annoying errors everywhere. I'd prefer to have slightly ugly code until we have a proper solution.

A proper solution, IMO, is going to be to have a way of saying "get me this asset of this module" for either server-side consumption. Important for moving modules to vendor and moving code out of the webroot.